consensus/src/consensus/inner/commit.rs

```diff
@@ -224,7 +224,7 @@ impl ConsensusInner {
         }
 
         // Check that all the transaction proofs verify
-        self.verify_transactions(block.transactions.iter())
+        self.verify_transactions(block.transactions.clone()).await
```

Comment:
Does this have to be `clone`d?

Reply:
`SerialBlock` is only borrowed in this context, so sadly - yes; however, the cost is nothing compared to blocking happening otherwise